.github/workflows/InvokeCI.yml

```diff
@@ -86,3 +86,19 @@ jobs:
       override_git_describe: ${{ inputs.override_git_describe }}
       git_ref: ${{ inputs.git_ref }}
       skip_tests: ${{ inputs.skip_tests }}
+
+  notify-odbc-main:
+    uses: ./.github/workflows/NotifyExternalRepositories.yml
+    secrets: inherit
+    if: ${{ inputs.git_ref == '' }}
```

Comment:
I think this (and the next) need a section like:
```yml
needs:
   - linux
   - osx
   - windows
   - etc
```
so that both those are run only AFTER the rest has completed.
I think as dependency anything that builds extensions, that should be: OSX, Windows, LinuxRelease, Python and R.